modules/oe_showcase_list_pages/config/install/pathauto.pattern.list_page_url_alias_pattern.yml

```
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: list_page_url_alias_pattern
+label: 'List page URL Alias Pattern'
+type: 'canonical_entities:node'
+pattern: 'list/[node:title]'
```

the ticket indicated "Add a simple url alias pattern that uses the title." I think that means:
```suggestion
pattern: '[node:title]'
```

---

Changed